worknenjoy/gitpay
# feat: Feedback after first task

## Description

This PR adds a feedback after a first task added

## Changes

Mostly on the task.js file

## Issue

> Closes #266 

## Impacted Area

> Task page

## Steps to test

You just need to create a task, its validated by the localStorage whether it should show or not the modal

- Create Task

## Before
![image](https://user-images.githubusercontent.com/38966428/79213076-f0a9cd00-7e1e-11ea-80b7-d64af4b9b53d.png)

## After
![image](https://user-images.githubusercontent.com/38966428/79213046-e5ef3800-7e1e-11ea-9d62-24198123ced3.png)
